hw_5/task2.py

```diff
@@ -0,0 +1,10 @@
+def sum(a, b):
+    if b == 0:
+        return a
+    else:
+        if b > 0:
+            return sum(a + 1, b - 1)
+        else:
+            return sum(a - 1, b + 1)
```

Comment:
Во-первых, по условию, `b` не может быть меньше нуля, Во-вторых, можно значительно упростить.
```suggestion

        return sum(a + 1, b - 1)
```